deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -598,6 +598,53 @@ spec:
             status:
               description: CephBlockPoolStatus represents the mirroring status of Ceph Storage Pool
               properties:
+                cephx:
+                  properties:
+                    daemon:
```

Comment:
CephBlockPool doesn't have a daemon, so we should make sure to use a type struct that doesn't include this. Probably a custom one will be needed